docker/production/entrypoint-sidekiq.sh

```
@@ -1,13 +1,16 @@
 #!/usr/bin/env bash
 cd /usr/src/app
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)